lib/acts_as_tenant.rb

```diff
@@ -109,6 +109,14 @@ def self.without_tenant(&block)
     self.current_tenant = old_tenant
     self.unscoped = old_unscoped
   end
+
+  def self.should_require_tenant?
+    if ActsAsTenant.configuration.require_tenant.respond_to?(:call)
```

Comment:
You're already inside the `ActsAsTenant` namespace, so you should be able to simplify this.
```suggestion
    if configuration.require_tenant.respond_to?(:call)
```